.github/workflows/ci.yml

```diff
@@ -48,7 +48,7 @@ jobs:
       run: npx relay-compiler
     - name: Typecheck frontend
       working-directory: frontend
-      run: npm run typecheck
+      run: tsc
```

Comment:
So you want to include the lib check in CI? This is an intentional change?

Reply:
It was in there before as well, if you look at the `typecheck` script before this PR. But yes, I think that's correct as well: We have `skipLibCheck` to speed up development, but that, at least in theory, opens us up to incorrect typechecking. In CI, where we have the time, we should prioritize correctness, IMO.